lib/internal/modules/cjs/loader.js

```diff
@@ -85,18 +84,17 @@ const {
 
 const isWindows = process.platform === 'win32';
 
+let requireDepth = 0;
+let statCache = new Map();
```

Comment:
☝️ Could initialize as `null` since below it will get inited as `new Map()`.

Reply:
I thought about that as well but I figured that having the main value in there makes it clearer what this variable normally contains.